actions/cloud.tsx

```diff
@@ -113,6 +113,18 @@ export function validateBusinessEmail(email = '') {
     };
 }
 
+export function validateWorkspaceBusinessEmail() {
+    trackEvent('api', 'api_validate_workspace_business_email');
+    return async () => {
```

Comment:
question: why are we returning a function here rather than doing the job directly? it kinda look weird to have to call `validateWorkspaceBusinessEmail()()`

Reply:
afaik is to facilitate the way we can execute this functions either calling them this way or dispatching them with Dispatch.

Reply:
Oh that makes sense 👍🏻